### PR TITLE
[ready 4 review] Add Kubernetes to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
- Add the kubernetes binaries to conda-forge

### CODE REVIEW

**Comment by sodre** [comment]
@isuruf, it looks like I caught you in a good time. Would you mind reviewing the Kubernetes PR and fakechroot #5368  as well?

File: recipes/kubernetes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  commands:
+    - hyperkube --version
+
+about:
+  home: https://www.kubernetes.io
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: Production-Grade Container Orchestration
+  doc_url: https://
```
Comment (isuruf) [inline recipes/kubernetes/meta.yaml]: Can you fix this with a commit title having `[skip ci] [ci skip]`?

File: recipes/kubernetes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - go 1.8.*
+    - rsync
+    - make
+    - toolchain
+  run:
```
Comment (isuruf) [inline recipes/kubernetes/meta.yaml]: Can you remove this while you are at it?

**Comment by sodre** [comment]
Kubernetes doc_url and "run:" were removed.

File: recipes/kubernetes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kubernetes/{{ name }}/archive/v{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [not linux]
```
Comment (isuruf) [inline recipes/kubernetes/meta.yaml]: osx doesn't work?

Comment (sodre) [reply recipes/kubernetes/meta.yaml]: Not for the hypercube build. I can investigate building the client side-only for OSX. Can I do that after the merge? I usually have better luck using the CI servers that way.

Comment (isuruf) [reply recipes/kubernetes/meta.yaml]: sure

**Comment by jakirkham** [comment]
Nice! Thanks @sodre.

**Pull request merged**